### DISCUSSION TITLE
is/will NvCloth support PhysX5.1?

### DISCUSSION THREAD

**Discussion by JailbreakPapa** [NVIDIA-Omniverse/PhysX#11] [Q&A] [answered]
title. im not exactly sure how 5.1 handles cloth so I'm wondering if NvCloth will add support.

**Comment by amoravanszky** [comment, marked as answer]
We do have position based dynamics cloth in this release, please check out SnippetPBDCloth: 

![image](https://user-images.githubusercontent.com/8350045/200789546-fe7e3e7e-d15a-4428-ba8e-1b29eb8e28c4.png)

We are also working on FEM based clothing but this is a problem with a lot of wrinkles so I cannot say when everything will be ironed out.  In any case any and all cloth tech we make will be part of PhysX, it will not be a separate library like NvCloth that is deprecated and no longer being worked on.
